src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public FigureType getRundomFigure() {
+        int index = new Random().nextInt(FigureType.values().length);
+        return FigureType.values()[index];

```

Comment:
we should return instance of some figure here(Circle, Square, ...) not FigureType